src/main/scala/org/sf1bacon/twitterextractor/Neo4jAPI.scala

```diff
@@ -55,13 +55,21 @@ object Neo4jAPI {
        |              r.verified = ${restaurant.verified},
        |              r.location = "${restaurant.location}",
        |              r.url = "${restaurant.url}",
-       |              r.added = "$modtime"
+       |              r.added = "$modtime",
+       |              r.googleID = "${restaurant.googleID}",
+       |              r.twitterID = "${restaurant.twitterID}",
+       |              r.lat = ${restaurant.latlong._1},
+       |              r.lon = ${restaurant.latlong._2}
```

Comment:
line 52: `"""MERGE (r:Restaurant { id: "${restaurant.username}", name: "${restaurant.name}" })`
be carefull here, some restaurant/user could remove account, and a new one can use this username with a different name.. allways MERGE only with UNIQUES inside the statement: `"""MERGE (r:Restaurant { id: "${restaurant.username}"})` .. `MERGE` evaluates all statement, and will try to create if both name and username dont match.. and it will broke if theres different name but same username. Non UNIQUEs should be assigned on `ON MATCH` and `ON CREATE`